udata/core/dataset/events.py

```diff
@@ -49,7 +49,8 @@ def publish(url, document, resource_id, action):
         "dataset_id": str(document.id),
         "document": resource,
     }
-    r = requests.post(url, json=payload)
+    headers = {"Authorization": f"Bearer {current_app.config['RESOURCES_ANALYSER_API_KEY']}"}
```

Comment:
I would add a `if current_app.config['RESOURCES_ANALYSER_API_KEY']`

Reply:
Sure!
I'm now wondering if we should have the `RESOURCES_ANALYSER_API_KEY` be `""` in the settings.py file... because right now it's set to `api_key_to_change`.

So unless someone sets it on purpose to `""`, they'll always send the auth header.

What do you think @maudetes?

Reply:
Yes indeed, agreed for changing it to an empty value!